Realm/RLMSyncUser_Private.hpp

```diff
@@ -48,8 +51,14 @@ class CocoaSyncUserContext : public SyncUserContext {
      paths (e.g. `/~/path/to/realm`).
      */
     std::unordered_map<std::string, RLMSyncSessionRefreshHandle *> m_refresh_handles;
-
     std::mutex m_mutex;
+
+    /**
+     An optional callback that can be used to inform an application should a request to
```

Comment:
"An optional callback invoked when the authentication server reports the user as being in an expired state."